### DISCUSSION TITLE
Can we view more than one iTwin in the same viewer?

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#6147] [Q&A]
Our customer will potentially have many iTwins that represent each logical section their state roadway network.  Is it possible to display the contents of more than one iTwin in a single viewer?

**Comment by pmconne** [comment]
Possible? Technically, yes. Well-supported? Currently, no. It's an area of active investigation.
Details of your use case(s) are welcome.
